include/inviwo/qt/editor/inviwomainwindow.h

```diff
@@ -247,8 +247,9 @@ class IVW_QTEDITOR_API InviwoMainWindow : public QMainWindow, public NetworkEdit
     ResourceManagerDockWidget* resourceManagerDockWidget_;
     PropertyListWidget* propertyListWidget_;
     HelpWidget* helpWidget_;
-    WelcomeWidget* welcomeWidget_ =
-        nullptr;  ///< Use delayed initialization as it can be expensive.
+
+    ///< Use delayed initialization as it can be expensive.
```

Comment:
Use `///` or `//!`. Using `///<` refers to the left of the comment.
```suggestion
    /// Use delayed initialization as it can be expensive.
```